proto/src/universe.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package universe;
+
+service Universe {
+    rpc CreateKeyspace (CreateKeyspaceRequest) returns (CreateKeyspaceResponse);
+    rpc ListKeyspaces (ListKeyspacesRequest) returns (ListKeyspacesResponse);
+}
+
+message Region {
+    string cloud = 1;
```

Comment:
Should we use an enum for `cloud`? https://protobuf.dev/programming-guides/proto3/#enum

Reply:
Is there a way to have a catch-all "Other" that is a generic string in the enum?

Reply:
Other than using `OTHER` as an enum value that's used when someone is deploying at on-prem or on a less known cloud, I don't think so. I was just thinking of a way to avoid having to validate it at every receiver of this message.